### DISCUSSION TITLE
Can i change the Height/Width of TaskBar?

### DISCUSSION THREAD

**Discussion by GoldyWang2013** [valinet/ExplorerPatcher#2781] [Q&A] [answered]
Change the Height of TaskBar when it is on bottom/Top,
and
Change the Width of TaskBar when it is on Side(Left/Right)

in windows10, i place TaskBar on right, and increase the width, so can see more caption info of every task icon...

**Comment by pyrates999** [comment]
Yes you can: https://www.wikihow.com/Alter-the-Size-of-Your-Windows-Desktop-Taskbar

**Reply by GoldyWang2013** [reply]
read  above url, and just tell us : there is no way to change the Taskbar size?

**Reply by pyrates999** [reply]
Please show a screen shot of what you mean then.

**Comment by GoldyWang2013** [comment]
![TaskBarSizing](https://github.com/valinet/ExplorerPatcher/assets/16570478/b060bb5c-5061-4dfc-b384-f9a67f189e88)

**Reply by GoldyWang2013** [reply]
Win11  installed ExplorerPatcher, can move the TaskBar to RightSide of Desktop,  but can not change the TaskBar size, which is really want by me...

**Reply by pyrates999** [reply]
Try moving it to the left side of the desktop.  I remember after right clicking on it and unticking lock the taskbar, I was able to resize it.

Did you try right clicking the taskbar and unticking **lock the taskbar**?  And then try resizing it?

**Comment by GoldyWang2013** [comment, marked as answer]
thanks, you are right, unticking lock the taskbar, i can resize it....
